libs/sdk-ui-kit/src/Header/tests/generateHeaderMenuItemsGroups.test.ts

```diff
@@ -144,9 +252,42 @@ describe("generateHeaderMenuItemsGroups", () => {
         ]);
     });
 
+    it("should return data item with datasource href and workspace in uri if platformEdition is free, hasNoDataSet is true and enableRenamingProjectToWorkspace is true", () => {
```

Comment:
Please add one more test case when `enableRenamingProjectToWorkspace` is `false`.

`it("should return data item with datasource href and project in uri if platformEdition is free, hasNoDataSet is true and enableRenamingProjectToWorkspace is false"`

Reply:
As the existing test can be considered for case when `enableRenamingProjectToWorkspace` is `false`, the comment is marked as resolved.